java/flight/src/main/java/org/apache/arrow/flight/FlightService.java

```diff
@@ -172,6 +172,8 @@ public void completed() {
         responseObserver.onNext(producer.acceptPut(makeContext(responseObserver), fs).call());
         responseObserver.onCompleted();
       } catch (Exception ex) {
+        logger.error("Failed to process custom put due to: " + ex.getMessage());
```

Comment:
`logger.error("...", ex);` should work and insert the stack trace in the log, no?

Reply:
@lihalite thanks for the good suggestion. I have revised the code accordingly.